Source/OxyPlot/Series/HeatMapSeries.cs

```diff
@@ -122,6 +122,21 @@ public HeatMapSeries()
         public bool Interpolate { get; set; }
 
         /// <summary>
+        /// Gets or sets a value indicating whether to use discrete rectangles when rendering (ignored if Interpolate is set to true). The default value is <c>false</c>.
+        /// </summary>
+        public bool RenderDiscreteRectangles { get; set; }
```

Comment:
What will happen if a 3rd method of rendering heat maps is implemented? Then a bool property will have to be marked obsolete, I think.

I think it is better to do one of the following
- use an enum
- create a new class
- fall back automatically to rectangle rendering if image rendering will not work (e.g. log axes) or is not supported on the current platform